client/src/context/quiz-context.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { createContext, useContext, useEffect, useReducer } from "react";
+import { getQuizData } from "../database/data";
+import { QuizContType, QuizState } from "./quiz-context.type";
+import { manageState } from "./quiz-reducer";
+export const initialStateValue: QuizState = {
+  score: 0,
+  currentQuesNumber: 1,
+  username: "",
+};
+export const QuizContext = createContext({});
+
+export const QuizProvider = ({ children }: any) => {
```

Comment:
- Instead of using `any` for `QuizProvider` prop use `ReactNode` as prop type.
- Refer [this document](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/basic_type_example#useful-react-prop-type-examples) for useful prop types.